### PR TITLE
get frontend.properties from s3

### PR DESCRIPTION
**Pull request by johnduffell**
For all those who had to copy and paste a file from google drive and hated it, now setup.sh will do it for you magically.

Further benefits include access control being managed through janus rather than by me manually.

@adamnfish 🎉

### CODE REVIEW

File: setup.sh [hunk 1]
```diff
@@ -58,22 +57,6 @@ region = eu-west-1" > "$path/$filename"
   fi
 }
 
-create_aws_credentials() {
```
Comment (johnduffell) [inline setup.sh]: this was pretty out of date anyway in these days of janus

File: setup.sh [hunk 2]
```diff
@@ -39,8 +39,7 @@ create_frontend_properties() {
       mkdir "$path"
     fi
 
-    touch "$path/$filename"
-    EXTRA_STEPS+=("Ask a colleague for frontend.properties and add the contents to $path/$filename")
+    aws s3 cp --profile frontend s3://aws-frontend-store/template-frontend.properties "$path/$filename"
```
Comment (jamespamplin) [inline setup.sh]: Is this bucket versioned? If not, can we make it please :smile:

Comment (jamespamplin) [inline setup.sh]: Also can we please have an "installed" check for the `aws` cli

Comment (johnduffell) [inline setup.sh]: already is versioned by the looks

Comment (johnduffell) [inline setup.sh]: ok done that installing the aws cli

**Comment by jamespamplin** [comment]
:+1: great idea, thanks for this

**Pull request merged**